package.json

```diff
@@ -58,8 +58,8 @@
     "cross-env": "^6.0.3",
     "delay": "^4.3.0",
     "dirty-chai": "^2.0.1",
-    "electron": "^7.1.11",
-    "electron-builder": "^22.3.2",
+    "electron": "^7.1.14",
```

Comment:
I did not update to 8.x because it does not support old school Tray icons on Linux (#1153) and properly addressing that requires time.

Reply:
<img width="482" alt="Screen Shot 2020-03-10 at 10 18 45 AM" src="https://user-images.githubusercontent.com/50103/76321656-b21d7180-62b8-11ea-9c16-0b20876f4be4.png">